### DISCUSSION TITLE
Additional indicators - QQE and Bollinger Percent B Indicator

### DISCUSSION THREAD

**Discussion by alfredo-catalano** [tarkah/tickrs#97] [Ideas]
Any support or interest for adding additional indicators? 

https://www.tradingview.com/support/solutions/43000501971-bollinger-bands-b-b/
https://www.tradingview.com/script/aCIR3UdM-Quantitative-Qualitative-Estimation-QQE/

Thank you

**Comment by tarkah** [comment]
Yeah, maybe in the future some of these can be added similar to how we have volumes displayed under the main graph, depending on complexity and data needed to recreate
